app/models/comment.rb

```diff
@@ -1,4 +1,9 @@
 class Comment < ApplicationRecord
-  belongs_to :author
-  belongs_to :post
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+  after_create :update_comment_count
+end
+
+def update_comment_count
+  post.increment!(:comments_counter)
```

Comment:
- It would be nice to use one of the Rails Active Callbacks - `after_save` to call the #update_comment_counter 😉